### PR TITLE
data:Ft4us5

### PR DESCRIPTION
**Pull request by Ft4us5**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3487"></a>

### CODE REVIEW

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Please review and validate your JSON data. 
Remove extra "," to rectify this or use online json validator tools

File: data/Ft4us5.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+  ],
+  "milestones":[
+    {
+      "title": "Won at ODCC challenge",
+      "date": "Jan 2023",
+      "color": "purple",
+      "description": "pitched solutions to the problem statements given to me by Bitbns"
+    }
+    ],
```
Comment (Tejender1521) [inline data/Ft4us5.json:36]: `]`

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Remove ", "

File: data/Ft4us5.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          "title": "Won at ODCC challenge",
+          "date": "Jan 2023",
+          "color": "purple",
+          "description": "pitched solutions to the problem statements given to me by Bitbns"
+        }
+        ]
+    }
+  
+  
+
```
Comment (Panquesito7) [inline data/Ft4us5.json:40]: Please remove these extra lines.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/Ft4us5.json (hunk 2 above)
Comment (Pradumnasaraf) [reply data/Ft4us5.json:40]: Hey, David. As mentioned previously, we don't need to point out extra lines fixes. We have a workflow set to fix that after the merge.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**